src/main/properties/annotators.properties

```diff
@@ -18,6 +18,14 @@ org.aksw.gerbil.annotators.definition.Aida.cacheable=true
 org.aksw.gerbil.annotators.definition.Aida.class=org.aksw.gerbil.annotator.impl.aida.AidaAnnotator
 org.aksw.gerbil.annotators.definition.Aida.constructorArgs=${org.aksw.gerbil.annotator.Aida.ServieURL}
 
+### ABACO
+org.aksw.gerbil.annotators.ABACO.serviceUrl=http://tec.citius.usc.es/gerbilendpoint/
```

Comment:
if https is available that would be nice to use. :)

Reply:
works with https.

Reply:
Changed it, thanks. :)